src/Equinor.ProCoSys.IPO.Command/EventHandlers/PostSaveEvents/IpoCompletedEmailHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Domain.Events.PostSave;
+using Equinor.ProCoSys.IPO.Email;
+using MediatR;
+using Microsoft.Extensions.Options;
+
+namespace Equinor.ProCoSys.IPO.Command.EventHandlers.PostSaveEvents
+{
+    public class IpoCompletedEmailHandler : INotificationHandler<IpoCompletedEvent>
+    {
+        private readonly IEmailService _emailService;
+        private readonly IOptionsMonitor<MeetingOptions> _meetingOptions;
+
+        public IpoCompletedEmailHandler(IEmailService emailService, IOptionsMonitor<MeetingOptions> meetingOptions)
+        {
+            _emailService = emailService;
+            _meetingOptions = meetingOptions;
+        } 
+
+        public Task Handle(IpoCompletedEvent notification, CancellationToken cancellationToken)
+        {
+            var constructionCompanyParticipant = notification.Invitation.Participants.SingleOrDefault(p =>
+                p.SortKey == 1 && p.Organization == Organization.ConstructionCompany);
+
+            if (constructionCompanyParticipant?.Email == null)
+            {
+                return Task.CompletedTask;
```

Comment:
Does this mean that if participant is a functional role and does not have a group email, no one will be notified? Should probably add a comment here as the logic is a little tricky.